### ISSUE TITLE
Article: [Why a11y Isn't Ironically Inaccessible]

### ISSUE THREAD

**Issue by faraixyz** [a11yproject/a11yproject.com#1162]
## Description

Sometimes people comment on how a11y is ironically inaccessible. That isn't the case since the abbreviation (known as a numeonym) is common (W3C, i18l) and it being like that actually helps curate resources (like this project) and community for those working on web accessibility. I guess this will be a good reference to link to when people point this out and can explain the name.

There are some posts which explain this already:

* [a11y is web accessibility](https://ericwbailey.design/writing/a11y-is-web-accessibility/)
* [A11y is not accessible](https://mobilea11y.com/blog/inaccessible-a11y/)
* [#a11y: Accessible or ironic?](https://tatianamac.com/posts/a11y-abbr/)

## Would you like to write the article or request that it be written?

I can write the post.

**Comment by scottaohara** [comment]
The a11yproject already has https://www.a11yproject.com/posts/2017-08-26-a11y-and-other-numeronyms/
